modules/openapi-generator/src/test/resources/3_0/issue6491.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: 3.0.1
+info:
+  title: My title
+  description: API under test
+  version: 1.0.7
+servers:
+  - url: https://localhost:9999/root
+paths:
+  /location:
+    get:
+      operationId: getLocation
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ArrayWithIneffectiveValidations'
+components:
+  schemas:
+    ArrayWithIneffectiveValidations:
+      type: array
+      items: {}
+      minProperties: 1
+      maxProperties: 5
+      minLength: 1
+      maxLength: 5
+      pattern: 'abcde'
+      multipleOf: 3
+      minimum: 1
+      maximum: 10
+      exclusiveMinimum: 0
+      exclusiveMaximum: 100
+
+    ObjectWithIneffectiveValidations:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+      minItems: 1
+      maxItems: 5
+      uniqueItems: true
+      minLength: 1
+      maxLength: 10
+      pattern: 'abcde'
+      multipleOf: 3
+      minimum: 1
+      maximum: 10
+      exclusiveMinimum: 1
+      exclusiveMaximum: 10
+
+
+    StringWithIneffectiveValidations:
+      type: string
+      minItems: 1
+      maxItems: 5
+      uniqueItems: true
+      minProperties: 1
+      maxProperties: 5
+      multipleOf: 3
+      minimum: 1
+      maximum: 10
+      exclusiveMinimum: 0
+      exclusiveMaximum: 100
+
+    IntegerWithIneffectiveValidations:
+      type: integer
+      minItems: 1
+      maxItems: 5
+      uniqueItems: true
+      minProperties: 1
+      maxProperties: 5
+      minLength: 1
+      maxLength: 10
+      pattern: 'abcde'
```

Comment:
How abut adding examples for:
- boolean schema
- null schema
- unset type (AnyType) schema